cmd/tgrade/root.go

```diff
@@ -318,6 +326,37 @@ func extendUnsafeResetAllCmd(rootCmd *cobra.Command) {
 	}
 }
 
+func areFeesTooHigh(cmd *cobra.Command) bool {
```

Comment:
Please add some doc like:
"prevent the common fat finger issue and reject tx with fees beyond a certain limit"